### ISSUE TITLE
Standalone Kubelet is no longer logging container info(“FieldPath” field) for event logs

### ISSUE THREAD

**Issue by diyi0926** [kubernetes/kubernetes#107440]
### What happened?

I used to use 1.171 standalone Kubelet mode without an API server for my cluster. Because I’m not using API server, I can only get container events from Kubelet logs.
But after I upgrade Kubelet to 1.20.1, I found that the log format is changed and Kubelet log is no longer logging container info(“FieldPath” field in logs) for events logs due to this change [https://github.com/kubernetes/kubernetes/pull/91712](url)
So with a newer version Kubelet, I cannot know which container those events belong to from Kubelet logs.

### What did you expect to happen?

container info(“FieldPath” field) should be added back to Kubelet event logs

### How can we reproduce it (as minimally and precisely as possible)?

Can reproduce it with any standalone Kubelet version after 1.18.20.

### Anything else we need to know?

I also want to what’s the best practice to get events for standalone Kubelet? Should “FieldPath” field add back to log? or is there any plan to add something like an “event server”, so users can configure and use it to get events for standalone Kubelet easier?

### Kubernetes version

1.20.1

### Cloud provider

N/A

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by pacoxu** [comment]
#91712 seems to only change the log style. 
```
I0604 09:34:41.839722   18865 event.go:278] "Event occurred" object="kube-system/kube-dns-c74654849" kind="ReplicaSet" apiVersion="apps/v1" type="Normal" reason="SuccessfulCreate" message="Created pod: kube-dns-c74654849-26248"
```
In this example, the pod name is still included in the log.

/cc @serathius
/sig instrumentation

**Comment by diyi0926** [comment]
> #91712 seems to only change the log style.
> 
> ```
> I0604 09:34:41.839722   18865 event.go:278] "Event occurred" object="kube-system/kube-dns-c74654849" kind="ReplicaSet" apiVersion="apps/v1" type="Normal" reason="SuccessfulCreate" message="Created pod: kube-dns-c74654849-26248"
> ```
> 
> In this example, the pod name is still included in the log.
> 
> /cc @serathius /sig instrumentation

Pod name is still there, but the "FieldPath" field which includes container name is missing
Old version log:
```
Event(v1.ObjectReference{Kind:"Pod", Namespace:"mirinfra", Name:"envoy-d62f499c881844e792e0e78df8266b5e000000", UID:"4cd8fe94f360fe56a9514923c9018ef0", APIVersion:"v1", ResourceVersion:"", FieldPath:"spec.containers{envoy-base-2}"}): type: 'Normal' reason: 'Killing' Stopping container envoy-base-2
```

**Comment by pacoxu** [comment]
@serathius would you confirm this is expected or a regression?

**Comment by diyi0926** [comment]
Gentle ping on this, this is a blocking issue, without a fix, we need to stay with v1.18. 
Do we have a plan to fix this? or maybe I can help to fix @serathius

**Comment by serathius** [comment]
Removing fields like "fieldPath" was not intended. It should be ok to reintroduce this field to logs like @pacoxu showed in https://github.com/kubernetes/kubernetes/issues/107440#issuecomment-1009786572

This is however not a bug as we don't provide any guarantee about log contents. Feel free to send a PR that will add this field back.
/kind feature
